src/components/TableCard/TableCard.jsx

```diff
@@ -160,7 +161,26 @@ const defaultProps = {
     downloadIconDescription: 'Download table content',
     severityLabel: 'Severity',
     emptyMessage: 'There is no data for this time range.',
+    // Card-specific labels needed for combo with table toolbar
```

Comment:
I think we wanna import these defaults from the Card component just to cut down on duplication

Reply:
Imported the default props and assigned to TableCard